### ISSUE TITLE
New term request: Abnormal respiratory conducting tube morphology

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#2556]
Label: Abnormal respiratory conducting tube morphology 

Parent class ID: HP_0002088 
Parent label: Abnormality of lung morphology 

Suggested text def: Any structural anomaly of the tubes of the respiratory system that allow passage of air from the trachea to the terminal bronchioles. 

Synonym(s): Abnormal conducting airways; Abnormal respiratory conducting zone; Abnormal respiratory passage 
Text def xref: MP:0004391

Logical def: has part' some (morphology and ('inheres in' some 'respiratory tube') and ('has modifier' some abnormal))

**Comment by cmungall** [comment]
I assume this is part of a project to fill in terms that are present in MP?

It's a good idea to test these terms before adding. Note that inference will give you different results than what is asserted in MP, currently:

```
    is_a MP:0001175 ! abnormal lung morphology
     is_a MP:0004391 ! abnormal respiratory conducting tube morphology  
      is_a MP:0002264 ! abnormal bronchus morphology
      is_a MP:0002267 ! abnormal bronchiole morphology
      is_a MP:0010856 ! dilated respiratory conducting tubes
```

the bronchus is not part of the lung in uberon, so this hierarchy would not be recapitulated.

HP is consistent with Uberon:

![image](https://user-images.githubusercontent.com/50745/30835540-89baa1d6-a20d-11e7-8cec-fc5f291d0665.png)

Uberon:

![image](https://user-images.githubusercontent.com/50745/30835596-c7a856e6-a20d-11e7-8bb4-12e6cdebb7fd.png)

TL;DR - use the reasoner

**Comment by pnrobinson** [comment]
I do not think that "respiratory conducting tube morphology" would sound very familiar to most MDs (I thinkl it means airways, but I have never heard the term before).

**Comment by nicolevasilevsky** [comment]
@pnrobinson Do we not want to add this term then?

This is for the MPO flyer terms, see ticket #2511 

Looks like the last comment on that ticket was to set up a Skype call- should I try to set up a call with us and Deanne?

**Comment by pnrobinson** [comment]
I think we need to figure out whether the corresponding human term makes sense. Abnormal airway morphology might be a good term. Need to figure out what we have now that might be close to it!

**Comment by cmungall** [comment]
I agree airway is a better label for a generic class

@nicolevasilevsky - What's the Fyler code for this term? I'm still not sure I understand the motivation for adding? I think having a consistent hierarchy with MP is generally a good thing but that seems a different project than Fyler

**Comment by nicolevasilevsky** [comment]
The flyer code for this term is Fyler:4232

**Comment by pnrobinson** [comment]
I am not sure that airway is 100% synonymous: The mouth and nose are the normal entry and exit ports for the airway. Entering air then passes through the back of the throat (pharynx) and continues through the voice box (larynx), down the trachea, to finally pass through the bronchi.
But I think that the current term Abnormality of the tracheobronchial system does this trick, and I do not think that the label above is helpful in human medicine. I am going to switch the label to "morphology" though.
